confidence/openfeature_provider.py

```diff
@@ -121,6 +121,24 @@ def resolve_boolean_details(
             flag_metadata=details.flag_metadata,
         )
 
+    async def resolve_boolean_details_async(
```

Comment:
The OpenFeature provider should not be used on it's own but rather together with the [OpenFeature SDK](https://github.com/open-feature/python-sdk/) so I would revert the changes in this class (and the test).

Reply:
Alright this has been removed, I'll look into adding the feature into OpenFeature SDK.

Reply:
I asked about it in their slack yesterday and was pointed to [this issue](https://github.com/open-feature/python-sdk/issues/284). Federico that has commented on that issue is one of the main contributors to the OpenFeature Python SDK and he seems positive 👍